project/dexter-vs/Dexter/Common/Tests/Utils/DexterSolutionManagerTest.cs

```diff
@@ -0,0 +1,195 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio;
+using Dexter.Common.Utils;
+
+namespace Dexter.Common.Tests.Utils
+{
+    [TestFixture]
+    public class DexterSolutionManagerTest
```

Comment:
These tests only check whether a ref variable `pfCancel` is assigned a value. They don't check anything useful at all.